Public/Src/Cache/ContentStore/App/Application.cs

```diff
@@ -285,11 +344,50 @@ private void Initialize()
 
             SetThreadPoolSizes();
 
+            string logFilePath = FileLog.GetLogFilePath(_logDirectoryPath, logFileBaseName: null, dateInFileName: true, processIdInFileName: true);
             if (_fileLog == null)
             {
-                _fileLog = new FileLog(_logDirectoryPath, null, _fileLogSeverity, _logAutoFlush, _logMaxFileSize, _logMaxFileCount);
+                _fileLog = new FileLog(logFilePath, _fileLogSeverity, _logAutoFlush, _logMaxFileSize, _logMaxFileCount);
                 _logger.AddLog(_fileLog);
             }
+
+            EnableRemoteTelemetryIfNeeded(logFilePath);
+        }
+
+        private void EnableRemoteTelemetryIfNeeded(string logFilePath)
+        {
+            if (!_enableRemoteTelemetry)
+            {
+                return;
+            }
+
+            if (_kustoUploader == null)
+            {
+                _logger.Warning

```

Comment:
>Warning [](start = 24, length = 7)

Why is this only a warning? This seems critical to me.

Reply:
It was mostly for testing purposes so that I could enable production CSV logs without uploading them to Kusto.

I'll make it an error.